versioned_docs/version-v2.18.x/user-guide/configuring-at-tls-for-zowe-server.md

```diff
@@ -572,30 +749,30 @@ TTLSConnectionAction ApimlNoX509ClientConnAction
   TTLSConnectionAdvancedParmsRef ZoweClientNoX509ConnAdvParms
 }
 
-# ConnectionAdvanced parameters for connections not requiring x.509 Client Certificate authentication
-# If the set Keyring has a default certificate this will not prevent sending it
-TTLSConnectionAdvancedParms ZoweClientNoX509ConnAdvParms
+# In case the connection requires a client certificate authentication, this is where the label is set for outbound connections.
+TTLSConnectionAdvancedParms ZoweClientX509ConnAdvParms
 {
+  CertificateLabel apimlcert
+  SecondaryMap Off
   SSLv3 Off
   TLSv1 Off
   TLSv1.1 Off
-  ApplicationControlled Off
-  CertificateLabel # Keep the Label empty to ensure a default certificate will not be picked from the keyring
-  SecondaryMap Off
   TLSv1.2 On
-  TLSv1.3 Off
+  TLSv1.3 On
 }
 
-# In case the connection requires a client certificate authentication, this is where the label is set for outbound connections.
-TTLSConnectionAdvancedParms ZoweClientX509ConnAdvParms
+# ConnectionAdvanced parameters for connections not requiring x.509 Client Certificate authentication
+# If the set Keyring has a default certificate this will not prevent sending it
```

Comment:
So, if there's a default cert and no CertificateLabel in AT-TLS it picks the default, right?